### DISCUSSION TITLE
on_air URL disconnecting on heavy image loading. How can I fix it?

### DISCUSSION THREAD

**Discussion by Anindya088** [zauberzeug/nicegui#1564] [Q&A] [answered]
### Question

Hi All,

I realized that if I am trying to use the ```on_air``` public URL, if my code has a heavy image then it auto-disconnects and creates a new URL again. Is there a way to fix this and allow the URL to load the image (it might take longer, but that is fine)
```python
from nicegui import ui, app

images = r'C:\apps\test\hivespace\app\images'
app.add_static_files('/images', images)
ui.image('/images/app/bg/sm/1.jpg')   # 8 mb image
```

In the console, there is no error, it just creates a new connection like this
![image](https://github.com/zauberzeug/nicegui/assets/65127974/410757b3-3f30-40c7-8ebb-34f287aaf228)

FYI... the image I was trying was around 8 mb.

Thanks,
Anindya

**Comment by rodja** [comment, marked as answer]
Oh, we have seen this limitation ourself but had not such a good reproduction. Thanks!  Will fix it in later releases. For now you'll need to provide a scaled down version of the image (which is generally recommended to reduce loading times).

**Reply by Anindya088** [reply]
Hi @rodja , thanks a lot for the update, sure will use scaled down versions for now :)
